File: C++/Niveau 2/2 - Découverte des tableaux/4 - Etude de marché.cpp
#include <iostream>
using namespace std;

int main(){

    int nombreProduits = 10, nombrePersonnes;

    cin >> nombreProduits >> nombrePersonnes;

    int produits[nombreProduits]= {0};

    for(int i = 1; i<= nombrePersonnes; i++){
        int numeroProduit;
        
        cin >> numeroProduit;

        produits[numeroProduit] +=  1;

    }

    for(int i = 0; i < nombreProduits; i++){
        cout << produits[i] << endl;
    }



} 

File: Python/Niveau 2/2 - Découverte des tableaux/4 - Etude de marché.py
nombre_produits = int(input())
nombre_personnes = int(input())

liste_produits_aimés = [0 for _ in range(nombre_produits)]

for x in range(nombre_personnes):
    numéro_produit = int(input())
    liste_produits_aimés[numéro_produit] += 1
for x in range(nombre_produits):
   print(liste_produits_aimés[x])